test/object-store/c_api/c_api.cpp

```diff
@@ -15,13 +15,15 @@
 #include <cstring>
 #include <numeric>
 #include <thread>
+#include <iostream>
```

Comment:
extra include here?

Reply:
Removed the debugging code, but forgot the header file.